packs/astral/actions/lib/BaseAction.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
made the fix to change BaseAction.py to action.py in just one commit 👍   Lesson learned that the GitHub web is not a good tool for making edit!